core/src/main/resources/hudson/model/View/sidepanel.jelly

```diff
@@ -60,7 +60,6 @@ THE SOFTWARE.
       <st:include page="sidepanel2.jelly" optional="true"/>
 
       <st:include page="tasks-bottom.jelly" it="${it.owner}" optional="true" />
-      <t:actions />
```

Comment:
Somehow this does not affect folder views. Which is good in that it means that contributed sidepanel links inside folders (**Credentials**, **Folder Support**, etc.) are still visible, but makes for an inconsistent experience compared to the root `Jenkins` dashboard view. In fact you have `RootAction`s (as badges) and folder `TransientActionFactory` actions (as sidepanel links) on the same page, which looks confusing to me.
![Screenshot from 2025-04-30 11-59-52](https://github.com/user-attachments/assets/3f68a1db-5248-40b8-89de-5dcb72886649)